OpenSim/Analyses/StaticOptimization.cpp

```diff
@@ -27,6 +27,7 @@
 //=============================================================================
 #include <iostream>
 #include <string>
+#include <memory>
```

Comment:
I think this should be included in the header instead, since we also use `unique_ptr` there.